src/lnp/application/payment/types.rs

```diff
@@ -262,13 +263,21 @@ impl DumbDefault for TempChannelId {
     }
 }
 
+#[derive(Wrapper, Clone, Debug, From, PartialEq, Eq)]
+pub struct RGBColor([u8; 3]);
+
+impl Strategy for RGBColor {
+    type Strategy = strategies::AsWrapped;
```

Comment:
no, this will not work this way; you need to manually implement StrictEncode/StrictDecode traits for the type

Reply:
Ok, writing them manually.